cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -64,10 +64,17 @@
         <meta property="og:title" content="{% block og_title %}{{ self.title() }}{% endblock %}">
         <meta property="og:type" content="{% block og_type %}website{% endblock %}">
         <meta property="og:url" content="{{ request.url }}">
+        {% if page and page.preview_image %}
+        {% set preview_image = image(page.preview_image, 'original') %}
         <meta property="og:image"
-              content="{{ static('nemo/_/img/logo.svg') }}">
+              content="{{ preview_image.url  }}">
+        {% else %}
+        <meta property="og:image"
+              content="{{request.scheme}}://{{request.get_host()}}{{ static('img/logo.png') }}">
+        {% endif %}
         <!--   Optional -->
-        <meta property="og:description" content="{% block og_desc %}{{- page.search_description if page and page.search_description -}}{% endblock %}">
+        <meta property="og:description" content="{%- block og_desc %}
+        {%- if page %}{{- page.search_description or page.seo_title -}}{% endif %}{% endblock %}">
```

Comment:
We still need one more fallback here. For example, the Blog index still has a blank `og:description` with this code.